packages/linux/package.mk

```diff
@@ -152,6 +152,9 @@ pre_make_target() {
 
     mkdir -p $ROOT/$PKG_BUILD/external-firmware/intel-ucode
       cp -a $(get_build_dir intel-ucode)/microcode.bin $ROOT/$PKG_BUILD/external-firmware/intel-ucode
+
+    FW_LIST="$(find $ROOT/$PKG_BUILD/external-firmware -type f \( -iname '*.bin' -o -iname '*.fw' \) | sed 's|.*external-firmware/||' | xargs)"
```

Comment:
`-type f` will miss the 4 firmwares referenced by symbolic links in the i915 directory.

Use `\( -type f -o -type l \)`, or `-not -type d` will also work.